src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -67,20 +83,20 @@ def create_launcher(fluent_launch_mode: LaunchMode = None, **kwargs):
         If an unknown Fluent launch mode is passed.
     """
     allowed_options = [mode for mode in LaunchMode]
-    if (
-        not isinstance(fluent_launch_mode, LaunchMode)
-        or fluent_launch_mode not in allowed_options
-    ):
+    if fluent_launch_mode not in allowed_options:
```

Comment:
I prefer not to create the above list. Simply the key needs to be a `LaunchMode`.

Reply:
We are using this list in higher level exception.

Reply:
Given that the enum type is fixed, I'm not sure that it's necessary.

Reply:
Done. Thank you.